weaviate/alias/getter.go

```diff
@@ -0,0 +1,49 @@
+package alias
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/except"
+)
+
+// Getter builder object to get a list of aliases
+type Getter struct {
+	connection *connection.Connection
+	className  string
+}
+
+// WithClassName specifies the class to which the alias belongs to
+func (s *Getter) WithClassName(className string) *Getter {
+	s.className = strings.TrimSpace(className)
+	return s
+}
+
+// Do get the list of alias
+func (s *Getter) Do(ctx context.Context) ([]Alias, error) {
+	return listAlias(ctx, s.connection, s.className)
+}
+
+func listAlias(ctx context.Context, conn *connection.Connection, className string) ([]Alias, error) {
```

Comment:
can we make this method a part of `Getter` struct?

Reply:
well. it doesn't have to be. Just simple func needing two arguments :). It doesn't need anything else from the struct. Even by making it part of struct, it makes sense to pass these args explicitly.